client/pkg/constants/constants.go

```diff
@@ -83,3 +83,6 @@ const CompressionThresholdBytes = 2048
 
 // InfraProviderHealthCheckInterval is the default interval of the infra provider health checks.
 const InfraProviderHealthCheckInterval = 30 * time.Second
+
+// MinTalosVersion is the minimum Talos version supported by Omni.
```

Comment:
I think we should clarify that it is the minimum for _new_ clusters

Reply:
no, that was meant to be for "new clusters only". so if you run 1.5 already, you can upgrade to 1.5.x any version